index.js

```diff
@@ -3,3 +3,4 @@ export {execaSync} from './lib/sync.js';
 export {execaCommand, execaCommandSync} from './lib/command.js';
 export {execaNode} from './lib/arguments/node.js';
 export {$} from './lib/script.js';
+export {ExecaError, ExecaSyncError} from './lib/return/cause.js';
```

Comment:
We could also call those `SubprocessError` and `SubprocessSyncError`. Not sure which one you prefer? Or any other name?

Reply:
I think the current names are the best. An error could potentially not be related to subprocesses too, like validation errors.

Reply:
Validation errors actually do not have `ExecaError` type, they are just `Error` instances. 

More precisely, we have 3 types of errors based on when they arise:
  1. Arguments validation, for example if the user does `execa(false)` or `execa(..., {cwd: false})`. Those throw synchronously an `Error` instance. 
  2. Exceptions by `child_process.spawn()`, i.e. the subprocess could not be spawned. Those are called "early errors" in the codebase. They reject asynchronously an `ExecaError` instance.
  3. Subprocess failures. They also reject asynchronously an `ExecaError` instance.

Ideally, the type `1.` would also reject an `ExecaError` instance, for consistency. I've tried to do implement this a few weeks ago and it was very difficult and complicated the codebase quite a lot, so I gave up on it.

But at the moment, `ExecaError` is always subprocess-related. That being said, that name still works for me. :)